docs/content/common-subs/kubestellar-apply-apache-openshift.md

```diff
@@ -109,15 +109,46 @@ subjects:
 - kind: ServiceAccount
   name: default
   namespace: my-namespace
+---
+apiVersion: v1
+kind: Service
```

Comment:
With this Service being used instead, you can and should remove the `hostPort: 8081` from the PodSpec in the Deployment above. That will remove an unnecessary detail and reduce the extra scc that needs to be added in OpenShift.